Modules/Devices/src/Devices.Infrastructure/PushNotifications/AzureNotificationHub/NotificationBuilder.cs

```diff
@@ -14,7 +14,6 @@ public static NotificationBuilder Create(NotificationPlatform platform)
     {
         NotificationBuilder builder = platform switch
         {
-            NotificationPlatform.Fcm => new FcmNotificationBuilder(),
```

Comment:
I'm going to use this thread to discuss some of the changes commented in this review.
The old `FcmNotificationBuilder` extends the `NotificationBuilder` class. This class is used to build a `Microsoft.Azure.NotificationHubs.Notification`. Hence the deletion: this does not make sense in an FCM context. Furthermore,
 - the `NotificationBuilder` class is in the `AzureNotification` namespace.
 - the equivalent of an AzureNotification Notification is an FCM (Multicast)**Message**, not an FCM **Notification**.

The way I see it, we have a few options here:
1. make the base class Build method return an object or dynamic, so that it can accomodate both these realities.
2. implement another class (be it `FcmMessageBuilder`) which, while sharing some similarities with `ApnsNotificationBuilder` (in the type of methods that are available), will return a totally different type -- what I did.
3. create another class/method which will receive a `Microsoft.Azure.NotificationHubs.Notification` and turn it into an Fcm Message.

Reply:
We can't just remove the support for AzureNotificationHub. 😲 

You should completely ignore the classes in the AzureNotificationHub folder and just create your own classes in your own folder/namespace. So this means: a new base class as well as new subclasses.

Reply:
Oh, so AzureNotificationHub is already working with FCM? I hadn't realised that. Very well, I believe I know how to right things now.